parlai/tasks/task_list.py

```diff
@@ -1018,4 +1018,16 @@
             "determine if the text is offensive or \'safe\'."
         ),
     },
+    {
+        "id": "MultiWOZ",
+        "display_name": "MultiWOZ",
+        "task": "multiwoz",
+        "tags": ["All", "ChitChat", "Goal"],
```

Comment:
I would actually remove the `ChitChat` spec here since this isn't really chitchat